File: zhiyoufy-web/src/model/dto/wms/WmsWorkerAppUserRelationFull.ts

import type { WmsWorkerAppUserRelation } from "./WmsWorkerAppUserRelation";


export interface WmsWorkerAppUserRelationFull extends WmsWorkerAppUserRelation {
  username: string;
}

export type WmsWorkerAppUserRelationParam = Omit<WmsWorkerAppUserRelationFull, "id">;

File: zhiyoufy-server-java/zhiyoufy-server/src/main/java/com/example/zhiyoufy/server/domain/dto/wms/WmsWorkerAppUserRelationFull.java
package com.example.zhiyoufy.server.domain.dto.wms;

import com.example.zhiyoufy.mbg.model.WmsWorkerAppUserRelation;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class WmsWorkerAppUserRelationFull extends WmsWorkerAppUserRelation {
	private String username;
}
